build.gradle

```diff
@@ -3,9 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+        google()
```

Comment:
I guess we also need to add the `google()` repo to the `L:16` as well? the sample project can't sync otherwise on my device (it could totally my laptop being weird)